frontend/app/src/people/utils/start_up_modal.tsx

```diff
@@ -241,14 +244,16 @@ const StartUpModal = ({ closeModal, dataObject, buttonColor }) => {
             width: '425px'
           }}
         >
-          {step === 1 ? <StepOne /> : step === 2 ? <StepTwo /> : <StepThree />}
+          {step === 1 ? (<StepOne />) : step === 2 ? (<StepTwo />) : (<StepThree />)}
         </EuiModal>
       </EuiOverlayMask>
     </>
   );
 };
 
-export default observer(StartUpModal);
+
+
+export default StartUpModal;
```

Comment:
is there a reason why we removed `observer`

Reply:
This may have occurred after merging with master.